runtime/sema/type.go

```diff
@@ -6264,3 +6264,17 @@ func getFieldNames(members []*Member) []string {
 
 	return fields
 }
+
+func isNumericSuperType(numberType Type) bool {
```

Comment:
Maybe this can be a property of `NumericType` instead? That way we would keep all the "properties" of a numeric type decoratively in one place.

Reply:
That's a good idea! Added with https://github.com/onflow/cadence/pull/1360/commits/b860ea00296045f10e413b5e3fcd0232471599fe